test/service/data/pfb_manifest.results.json

```diff
@@ -264,7 +264,7 @@
         "relations": []
     },
     {
-        "id": "dcb224a5-a6a3-550b-8215-3bd21e276fa1",
+        "id": "3287e942-6014-5c13-b254-d0e1b826843f",
```

Comment:
Like this one.

Reply:
The id's change because https://github.com/DataBiosphere/azul/blob/31e7003a080d4a089dc5a7066f8071ff97df86e3/src/azul/service/avro_pfb.py#L142 (You might have to click the link to read the full comment).

Reply:
Hmm, yeah, that doesn't work. We should use the `document_id`. We recently changed the way samples are represented so the uniqueness concern may not apply anymore. I'd rather omit problematic entities than to create a whole new set of identifiers for all entities.

Reply:
After further thought and inspection, using `document_id` won't work. It really needs to be something content addressable.